### PR TITLE
Update database to version 1.3.9

### LINKED ISSUE #84: Upgrade database to v1.3.9
https://github.com/boolder-org/boolder-data/releases/tag/v1.3.9

### PR DESCRIPTION
**Pull request by wang-li**
Integrate the new version of the boolder database.

Resolves #84

### CODE REVIEW

**Review by nmondollot — APPROVED** [review]

**Comment by nmondollot** [comment]
FYI I have an uncommitted file appearing in `/Users/nico/Dev/boolder-android/app/schemas/com.boolder.boolder.data.database.BoolderAppDatabase/16.json`

Content:
https://gist.github.com/nmondollot/d0f4c1dd138351202f82deb0e05104fe

First time I'm seeing this, is it normal? Should it be committed inside the repo?

**Comment by wang-li** [comment]
@nmondollot Yes, this is normal. It is the database schema that is generated by Room upon compilation, in case we want to perform migrations across the different versions of the database. This is useful for the user's database (which contains the ticked problems and the projects, and you'll see that the generated `.json` schema has been commited), but not for the boulders one, as this database is always initialized from the `.db` file.

In short, the schema under `app/schemas/com.boolder.boolder.data.userdatabase.UserDatabase` have to be versioned, but not the ones under `app/schemas/com.boolder.boolder.data.database.BoolderAppDatabase` (at least until we continue to initialize from a `.db` file)
